lib/__tests__/__snapshots__/transform.js.snap

```diff
@@ -240,3 +240,339 @@ foo
 </span>
 =========="
 `;
+
+exports[`native components basic 1`] = `
+"==========
+
+      export default class FooComponent extends Component {
+      }
+  
+~~~~~~~~~~
+foo
+~~~~~~~~~~
+ => tagName: div
+~~~~~~~~~~
+
+      import { tagName } from \\"@ember-decorators/component\\";
+      @tagName(\\"\\")
+      export default class FooComponent extends Component {
```

Comment:
we might have to check where `Component` comes from. otherwise we risk applying the codemod to glimmer components, which already have the outerHTML behavior

Reply:
Good catch! Will work on this.

Reply:
Implemented with the last commit!